src/renderers/react/ElementWrapper.tsx

```diff
@@ -32,6 +32,7 @@ const Body = styled("div")`
       opacity: 1;
     }
   }
+  min-height: 134px;
```

Comment:
It'd be nice to avoid hardcoding this – but that'll need a refactor that probably belongs in a separate PR, so we're fine for now!

Reply:
👍